### PR TITLE
Re-organized files in the repo

### PR DESCRIPTION
**Pull request by robfairh**
Currently, the files have an order hard to follow.
I moved things around and the organization makes more sense now.

I also deleted a bunch of files no longer necessary.

### CODE REVIEW

**Comment by samgdotson** [comment]
What command did you use to reorganize the files? 

``mv <file> <to-dir>`` or ``git mv <file> <to-dir>``? If you just did mv, then then you will lose file history. ``git mv`` preserves history and should be used.

File: README.md [hunk 1]
```diff
@@ -1,26 +1,3 @@
...
-* figures/: holds the figures with the results from serpent
-
-* docs/: holds the latex files
-
-``auxiliary.py`` does the following:
-- Adds legends to serpent geometry figures.
-- Plots energy spectrum.
-- Plots flux in a single detector.
-- Plots fluxes in bottom reflector, core, and top reflector.
+This repository holds the necessary files to model the MHTGR\350.
```
Comment (samgdotson) [inline README.md:3]: Did you mean for this README to be so short?

**Review by samgdotson — APPROVED** [review]

**Comment by samgdotson** [comment]
Seems fine, but the repo README should have more information.

File: README.md (hunk 1 above)
Comment (robfairh) [reply README.md:3]: There are a bunch of PRs coming in, I will add a better description in the near future.
I didn't want to do everything on this one to try to keep it as small as possible.
